src/components/chats/components/CreateNewChat/index.tsx

```diff
@@ -0,0 +1,21 @@
+import { Skeleton } from '@/components/ui/skeleton'
```

Comment:
`index.tsx` => We're avoiding it like the plague. The only exception if for defining routes in the /pages folder. Why? Bad DX. It makes the IDE super hard to operate with multiple open files.
Convention (open to changes!):
- Main file in the components "root".
- A Folder named like the root file.
- Dependencies inside the folder
-
CreateNewChat.tsx
[Folder CreateNewChat]
    CreateChat.tsx
    createNewChatHooks.ts
    createNewChatTypes.ts
    
Note in the folder example the verbosity of the hooks and types files. Again, by design to easily identify the files in the IDE.